TASK: Migrate fusion `context.currentRenderingMode` statements

mficzel opened the pull request:
Statements accessing using`(node|documentNode|site).context.currentRenderingMode.(edit|preview|name|title|fusionPath|options)` are migrated. Other uses of `context.currentRenderingMode` are marked with a comment.

Resolves: #23

ahaeslich reviewed (changes requested):
The new rule should also be added to `config/set/contentrepository-90.php`. If think this commented lines could fit: https://github.com/neos/rector/blob/24d8ffcb3583dfef370ba5736ac3e37cd3a8eaaa/config/set/contentrepository-90.php#L290-L292

dlubitz reviewed (commented):
Thank you for taking care. 

Look great, but Anke is right.

mficzel commented:
Offcourse @ahaeslich is right. Added the line and also adjusted the comments for the other renderingMode rectors.

Any suggestions how to adjust this php wise. How can we migrate `ContentContext::getCurrentRenderingMode` >> `RenderingModeService::findByCurrentUser`

ahaeslich commented:
@mficzel do you want to address the php transformation in this PR too?

> How can we migrate ContentContext::getCurrentRenderingMode >> RenderingModeService::findByCurrentUser
Just to be clear: Are you asking how to write such a PHP Transformation? Like this one: https://github.com/neos/rector/pull/17?

mficzel commented:
@ahaeslich yes wanted to know wether we could migrate php code aswell
